### ISSUE TITLE
Reproduce template

### ISSUE THREAD

**Issue by jcohenadad** [spinalcordtoolbox/pediatric-template#8]
After #5 #6 #7 is done, we should try to re-generate the template for reproducibility purpose.

**Comment by rohanbanerjee** [comment]
I am following the steps from the `README` on the `nb/add_pediatriac_template` branch (including the specific SCT version mentioned). While straightening the SC of the subjects, I am facing issues for the subject `sub-103`. The following is the error traceback:


<img width="342" alt="Screen Shot 2023-10-02 at 12 04 50 PM" src="https://github.com/spinalcordtoolbox/pediatric-template/assets/25586344/96e290b7-2a0a-49ff-89be-691e84d27401">


Which leads to:

```
Traceback (most recent call last):
  File "/home/GRAMES.POLYMTL.CA/robana/spinalcordtoolbox/spinalcordtoolbox/scripts/sct_straighten_spinalcord.py", line 278, in <module>
    main(sys.argv[1:])
  File "/home/GRAMES.POLYMTL.CA/robana/spinalcordtoolbox/spinalcordtoolbox/scripts/sct_straighten_spinalcord.py", line 261, in main
    fname_straight = sc_straight.straighten()
  File "/home/GRAMES.POLYMTL.CA/robana/spinalcordtoolbox/spinalcordtoolbox/straightening.py", line 155, in straighten
    _, arr_ctl_phys, arr_ctl_der_phys, _ = get_centerline(img_ctl, self.param_centerline,
  File "/home/GRAMES.POLYMTL.CA/robana/spinalcordtoolbox/spinalcordtoolbox/centerline/core.py", line 184, in get_centerline
    im_centerline.data[np.clip(x_centerline_fit.round().astype(int), 0, im_centerline.data.shape[0] - 1),
IndexError: shape mismatch: indexing arrays could not be broadcast together with shapes (519,) (519,) (520,)
```

@NadiaBlostein did you ever face this issue?

**Comment by NadiaBlostein** [comment]
Yup! I thought it was resolved, will get back to you asap (may be over the weekend as my current academic program is very intensive).

**Comment by NadiaBlostein** [comment]
I think it may be related to the pediatric dataset, so once there is a final version up on git annex that should fix things, will keep you posted @rohanbanerjee

**Comment by NadiaBlostein** [comment]
@rohanbanerjee Yes! Turns out that something was not updated correctly for the dataset on git-annex. I just fixed it and updated the README above such that you obtain the correct version of the dataset.

**for more information:** For sub-103 and sub-109 T1w images, the spinal cord masks were not great so I did not keep them (such that future users do not just blindly use those labels for their projects). Within the context of the template pipeline, this means that the centerline would get generated automatically instead of the pipeline trying to use the faulty T1w masks to straighten the sub-103 and sub-109 images. Let me know if you have any more issues.
